src/styles/main.scss

```diff
@@ -2,4 +2,5 @@
 @import "./variables";
 @import "./generalSettings";
 @import "./resetStyles";
-@import "./buttons";
\ No newline at end of file
+@import "./buttons";
+@import"./mixins.scss";
```

Comment:
Drop `.sccs` from import and add new line

Reply:
Done